pilot/pkg/bootstrap/server.go

```diff
@@ -557,6 +565,13 @@ func (s *Server) initMCPConfigController(args *PilotArgs) error {
 			configStores = append(configStores, configController)
 			continue
 		}
+		if url.Scheme == mcpScheme || url.Scheme == mcpsScheme {
+			if url.Host == "" {
+				cancel()
+				return fmt.Errorf("invalid mcp server address %s", configSource.Address)
+			}
+			configSource.Address = url.Host
```

Comment:
So if it's mcp address you take only the host (no port) and overwrite the configSource.Address with that host? Is it the intention here?
Is port fixed value?

Reply:
No, if configSource.Address == ""mcp://127.0.0.1:9901", url.Host = "127.0.0.1:9901", it will only remove the scheme prefix: mcp://